routes/contacts.js

```diff
@@ -0,0 +1,120 @@
+const nanoid = require("nanoid");
+const express = require('express')
+const router = express.Router()
+const contactStorage = require("../models/contacts.json");
+const { Contact } = require("../models/contacts.js");
+const { contactSchema } = require("../models/contacts");
+ 
+const listContacts = () => {
+  return contactStorage;
+};
+
+const getById = (contactId) => {
+  return contactStorage.find((element) => element.id === contactId);
+};
+
+const removeContact = (contactId) => {
+  const index = contactStorage.findIndex((element) => element.id === contactId);
+  if (index > -1) {
+    contactStorage.splice(index, 1);
+    return true;
+  }
+  return false;
+};
+
+const addContact = (body) => {
+  const newId = nanoid;
```

Comment:
Teraz wywołujesz obiekt a nie funkcję, zobacz sobie w dokumentacji nanoid. To powinno wyglądać tak:
`const newId = nanoid.nanoid();`

Reply:
ok poprawione